dds/DCPS/Util.h

```diff
@@ -1,13 +1,13 @@
 /*
- *
- *
  * Distributed under the OpenDDS License.
  * See: http://www.opendds.org/license.html
  */
 
 #ifndef OPENDDS_DCPS_UTIL_H
 #define OPENDDS_DCPS_UTIL_H
 
+#include <tao/Basic_Types.h>
```

Comment:
Why is this needed?

Reply:
For this: https://github.com/objectcomputing/OpenDDS/blob/e043b21ee296e67060dabc8c69a0d43fb760e88b/dds/DCPS/Util.h#L138

Reply:
Ok, I was looking at the `array_count` function template -- the only other part of this file that changed